projects/js-packages/components/components/scan-report/index.tsx

```diff
@@ -26,12 +27,13 @@ import styles from './styles.module.scss';
  * DataViews component for displaying a scan report.
  *
  * @param {object}   props                   - Component props.
+ * @param {boolean}  props.hasPlan           - Whether the user has a plan.
  * @param {Array}    props.data              - Scan report data.
  * @param {Function} props.onChangeSelection - Callback function run when an item is selected.
  *
  * @return {JSX.Element} The ScanReport component.
  */
-export default function ScanReport( { data, onChangeSelection } ): JSX.Element {
+export default function ScanReport( { hasPlan, data, onChangeSelection } ): JSX.Element {
```

Comment:
Perhaps we can approach this differently, but this was added because it seemed to be the only way to account for the variation in Tooltip text when an item is `checked` but has no threats.

Reply:
If we need this, I would suggest detaching the component from the "user has a plan" business logic. If the root requirement at the component level is to determine whether to use a "threat" or "vulnerability" term, potentially we could use a `dataSource: 'protect_report' : 'scan_api'` prop fed from the scan status query hook in Protect. WDYT?

Reply:
>we could use a dataSource: 'protect_report' : 'scan_api' prop fed from the scan status query hook in Protect. WDYT?

Yep, great idea - will give it a shot!